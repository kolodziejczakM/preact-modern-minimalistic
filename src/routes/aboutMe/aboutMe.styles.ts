import { h } from 'preact';
import { styled, setPragma } from 'goober';
setPragma(h);

export const Wrapper = styled('section')``;
