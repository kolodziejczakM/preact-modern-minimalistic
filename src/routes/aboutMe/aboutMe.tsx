import { h } from 'preact';
import { TypedComponent } from '../../shared/typings/prop-types';

export const AboutMe: TypedComponent<{}> = () => <span>About me page</span>;
