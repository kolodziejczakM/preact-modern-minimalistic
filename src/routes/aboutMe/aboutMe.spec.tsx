import '@testing-library/jest-dom/extend-expect';
import { h } from 'preact';
import { render } from '@testing-library/preact';
import { AboutMe } from './aboutMe';

describe('AboutMe', () => {
    it('should render correctly', () => {
        const { asFragment } = render(<AboutMe />);

        expect(asFragment()).toMatchSnapshot();
    });
});
