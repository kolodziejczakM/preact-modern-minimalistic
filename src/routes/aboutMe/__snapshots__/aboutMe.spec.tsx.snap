// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutMe should render correctly 1`] = `
<DocumentFragment>
  <span>
    About me page
  </span>
</DocumentFragment>
`;
